data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -506,50 +401,32 @@ private void handleTokTab(ASTNode nextToDestinationNode) {
 			node = (ASTNode) (nextToDestinationNode.getChild(0));
 		else
 			node  = nextToDestinationNode;
-//		String table_name = (nextToDestinationNode.getChildCount() == 1) ?
-//				getUnescapedName((ASTNode) nextToDestinationNode.getChild(0)) :
-//				getUnescapedName((ASTNode) nextToDestinationNode.getChild(0)) +
-//						"." + getUnescapedName((ASTNode) nextToDestinationNode.getChild(1));
 		String table_name = (node.getChildCount() == 1) ?
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)